// Licensed to the Apache Software Foundation (ASF) under one
// or more contributor license agreements.  See the NOTICE file
// distributed with this work for additional information
// regarding copyright ownership.  The ASF licenses this file
// to you under the Apache License, Version 2.0 (the
// "License"); you may not use this file except in compliance
// with the License.  You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.

syntax="proto2";
package brpc;

option cc_generic_services = true;
option java_generic_services = true;
option java_package = "com.brpc";
option java_outer_classname = "GetFavicon";

message GetFaviconRequest {}
message GetFaviconResponse {}

// Hack for accessing /favicon.ico
service ico {
    rpc default_method(GetFaviconRequest) returns (GetFaviconResponse);
}
